tools/db_bench_tool.cc

```diff
@@ -4592,6 +4593,12 @@ void VerifyDBFromDB(std::string& truth_db_name) {
         abort();
       }
 
+      if (thread->shared->write_rate_limiter) {
+        thread->shared->write_rate_limiter->Request(
+            key.size() + value_size_, Env::IO_HIGH, nullptr /*stats*/,
+            RateLimiter::OpType::kWrite);
+      }
```

Comment:
nah, forget it. I'm wrong. Digging further I realized that all the things I mentioned above are already handled in `RunBenchmark` function.